src/role-access/containers/delete-exclusion/delete-exclusion.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { Answer, ExclusionNavigationEvent } from '../../models';
+import { AnswerHeaderText, AnswerLabelText } from '../../models/enums';
+
+@Component({
+  selector: 'exui-delete-exclusion',
+  templateUrl: './delete-exclusion.component.html'
+})
+export class DeleteExclusionComponent implements OnInit {
+
+  @Output() public eventTrigger = new EventEmitter();
+
+  public exclusionNavigationEvent = ExclusionNavigationEvent;
+  public answers: Answer[] = [];
+  public caption = AnswerHeaderText.DeleteExclusion;
+  public heading = AnswerHeaderText.CheckAnswers;
+  public hint = AnswerHeaderText.CheckInformation;
+
+  constructor(private readonly route: ActivatedRoute) {}
+
+  public ngOnInit(): void {
+    // Get the role exclusions from the route, which will have been put there by the resolver.
+    const roleExclusions = this.route.snapshot.data.roleExclusions;
+    this.populateAnswers(roleExclusions);
+  }
+
+  public onEventTrigger(event: ExclusionNavigationEvent) {
+    this.eventTrigger.emit(event);
+  }
+
+  private populateAnswers(exclusions: any[]) {
```

Comment:
Could any[] change to be Exclusions or similar? Void return type for this as well?

Reply:
@connorpgpmcelroy - you're right... I'll fix both of these